templates/openwisp2/settings.py

```diff
@@ -280,6 +280,7 @@
 {% if openwisp2_monitoring and openwisp2_celery_monitoring %}
     # monitoring checks are executed in a dedicated "monitoring" queue
     'openwisp_monitoring.check.tasks.perform_check': {'queue': 'monitoring'},
+    'openwisp_monitoring.monitoring.tasks.migrate_timeseries_database': {'queue': 'influxdb_migration'},
```

Comment:
I think this should be "monitoring", we don't have a queue called `influxdb_migration` here, must be a leftover.